tests/test_mux.py

```diff
@@ -1478,6 +1478,27 @@ def test_neighbor_miss_no_peer(
         for ip in test_ips:
             self.check_neighbor_state(dvs, dvs_route, ip, expect_route=False)
 
+    def test_warm_boot_mux_state(
+            self, dvs, appdb, dvs_route, setup_vlan, setup_mux_cable, setup_tunnel,
+            remove_peer_switch, neighbor_cleanup, testlog
+    ):
+        """
+        test mux initialization during warm boot.
+        """
+        self.set_mux_state(appdb, "Ethernet0", "active")
```

Comment:
To be consistent with warmboot expectations, we should test a scenario where warmboot is applicable in dualtor systems - when there is a mix of active and standby states.

Hence, please add a mix (some active, some standby) before restarting swss, and verify 1) that respective states remain unchanged after restarting swss in warm_restart mode. 2) There are no flaps -- avoiding false negatives where double flap can result in same state as before.

Reply:
Thanks Vaibhav, I'll be closing the PR and reopening one for master branch first then requesting cherry-pick for 202205. I'll copy and paste your feedback so that it doesn't get lost in the transition.